moirae/estimators/online/utils/model.py

```diff
@@ -57,18 +80,158 @@ def __init__(self,
         self.transients = transients
         self.cell_model = cell_model
         self.asoh = asoh
-        self.input_template = input_template
+        self.inputs = inputs
 
         # Capture the shape of the outputs
-        self._num_output_dimensions = self.cell_model.calculate_terminal_voltage(self.input_template,
-                                                                                 self.transients,
-                                                                                 self.asoh).to_numpy().shape[1]
+        self._num_output_dimensions = self.cell_model.calculate_terminal_voltage(new_inputs=self.inputs,
+                                                                                 transient_state=self.transients,
+                                                                                 asoh=self.asoh).to_numpy().shape[1]
 
     @property
     def num_output_dimensions(self) -> int:
         return self._num_output_dimensions
 
 
+class CellModelWrapper(BaseCellWrapper):

```

Comment:
Could you remove the duplicated text ("Base link between...") from the docstring in this class and the next? I've set up our documentation to show the relationship between classes, which makes it quick to see that CellModelWrapper is a type of BaseCellWrapper that is the "Base link between..."

Reply:
Done!